### ISSUE TITLE
Regenerate SSL certificates (which are issued by letsencrypt) OpenDJ 4.5.4 SSLException: SSLEngine is CLOSED

### ISSUE THREAD

**Issue by esahin18** [OpenIdentityPlatform/OpenDJ#294]
After upgrading to OpenDJ 4.5.4, I get the `SSLException: SSLEngine is CLOSED` error every few days (not frequently). I looked through the logs but couldn't find any output that could fix the problem.

This error does not cause the OpenDJ service to stop. OpenDJ service continues to operate properly.

**Things i have tried:**
* Modified the `config.ldif`
* Clean install OpenDJ 4.5.4 & Migrate OpenDJ from the old version (2.6.0)
* Regenerate SSL certificates (which are issued by letsencrypt)

**/opt/opendj/logs/errors:**
```
[08/Aug/2023:08:18:15 +0300] category=CORE severity=ERROR msgID=140 msg=An uncaught exception during processing for thread Worker Thread 7 has caused it to terminate abnormally. The stack trace for that exception is: SSLException: SSLEngine is CLOSED (SSLConnectionContext.java:293 SSLConnectionContext.java:224 SSLBaseFilter.java:457 SSLBaseFilter.java:365 SSLFilter.java:139 ExecutorResolver.java:87 DefaultFilterChain.java:260 DefaultFilterChain.java:177 DefaultFilterChain.java:109 DefaultFilterChain.java:88 ProcessorExecutor.java:53 DefaultFilterChain.java:390 NIOConnection.java:427 NIOConnection.java:401 LDAPServerFilter.java:614 LDAPServerFilter.java:533 LDAPClientConnection2.java:658 LDAPClientConnection2.java:239 LDAPServerFilter.java:589 LDAPServerFilter.java:243 LDAPServerFilter.java:199 ...)
[08/Aug/2023:08:18:15 +0300] category=CORE severity=NOTICE msgID=139 msg=The Directory Server has sent an alert notification generated by class org.opends.server.api.DirectoryThread (alert type org.opends.server.UncaughtException, alert ID org.opends.messages.core-140): An uncaught exception during processing for thread Worker Thread 7 has caused it to terminate abnormally. The stack trace for that exception is: SSLException: SSLEngine is CLOSED (SSLConnectionContext.java:293 SSLConnectionContext.java:224 SSLBaseFilter.java:457 SSLBaseFilter.java:365 SSLFilter.java:139 ExecutorResolver.java:87 DefaultFilterChain.java:260 DefaultFilterChain.java:177 DefaultFilterChain.java:109 DefaultFilterChain.java:88 ProcessorExecutor.java:53 DefaultFilterChain.java:390 NIOConnection.java:427 NIOConnection.java:401 LDAPServerFilter.java:614 LDAPServerFilter.java:533 LDAPClientConnection2.java:658 LDAPClientConnection2.java:239 LDAPServerFilter.java:589 LDAPServerFilter.java:243 LDAPServerFilter.java:199 ...)
```

```
● opendj.service - LSB: This is the opendj daemon
     Loaded: loaded (/etc/init.d/opendj; generated)
     Active: active (running) since Tue 2023-08-08 13:44:29 +03; 49min ago
       Docs: man:systemd-sysv-generator(8)
      Tasks: 130 (limit: 9492)
     Memory: 2.7G
        CPU: 14min 9.727s
     CGroup: /system.slice/opendj.service
             └─3609731 /usr/bin/java -server -Xms4g -Xmx4g --add-exports java.base/sun.security.x509=ALL-UNNAMED --add-exports java.base/sun.security.tools.keytool=ALL-UNNAMED -Dorg.opends.server.scriptName=start-ds org.opends.server.core.DirectoryServer --configFile /opt/opendj/config/config.ldif --quiet

Ağu 08 13:44:15 goldsgym systemd[1]: Starting LSB: This is the opendj daemon...
Ağu 08 13:44:29 goldsgym opendj[3609653]: Starting opendj: > SUCCESS.
Ağu 08 13:44:29 goldsgym systemd[1]: Started LSB: This is the opendj daemon.
```

```
          --- Server Status ---
Server Run Status:        Started
Open Connections:         93

          --- Server Details ---
Host Name:                <hostname>
Administrative Users:     <admin-user>
Administrative Users:     <admin-user>
Installation Path:        /opt/opendj
Version:                  OpenDJ Server 4.5.4
Java Version:             11.0.18
Administration Connector: Port 4444 (LDAPS)

          --- Connection Handlers ---
Address:Port : Protocol    : State
-------------:-------------:---------
--           : LDIF        : Disabled
8989         : Replication : Enabled
0.0.0.0:389  : LDAP        : Disabled
0.0.0.0:636  : LDAPS       : Enabled
0.0.0.0:1689 : JMX         : Disabled
0.0.0.0:8080 : HTTP        : Disabled

          --- Data Sources ---
Base DN:                      <base-dn>
Backend ID:                   userRoot
Entries:                      3477
Replication:                  Enabled
Missing Changes:              <not available>
Age of Oldest Missing Change: <not available>
```

```
ii  default-jre                          2:1.11-72                      amd64        Standard Java or Java compatible Runtime
ii  default-jre-headless                 2:1.11-72                      amd64        Standard Java or Java compatible Runtime (headless)
ii  opendj                               4.5.4                          all          OpenDJ
ii  openjdk-11-jre:amd64                 11.0.18+10-1~deb11u1           amd64        OpenJDK Java runtime, using Hotspot JIT
ii  openjdk-11-jre-headless:amd64   
```

**Comment by vharseko** [comment]
please check 4.5.5

**Comment by vharseko** [comment]
fixed [4.7.0](https://github.com/OpenIdentityPlatform/OpenDJ/releases/tag/4.7.0)
